tensorflow/core/kernels/ragged_tensor_to_tensor_op.cc

```diff
@@ -154,10 +157,10 @@ class RaggedTensorToTensorBaseOp : public OpKernel {
 
     TensorShapeProto shape_proto;
     {
-      PartialTensorShape partial_tensor_shape;
-      TF_RETURN_IF_ERROR(TensorShapeFromTensor(c->input(kShapeInputIndex),
-                                               &partial_tensor_shape));
-      partial_tensor_shape.AsProto(&shape_proto);
+        PartialTensorShape partial_tensor_shape;
+        TF_RETURN_IF_ERROR(TensorShapeFromTensor(c->input(kShapeInputIndex),
+                                                &partial_tensor_shape));
+        partial_tensor_shape.AsProto(&shape_proto);
```

Comment:
```suggestion
      PartialTensorShape partial_tensor_shape;
      TF_RETURN_IF_ERROR(TensorShapeFromTensor(c->input(kShapeInputIndex),
                                               &partial_tensor_shape));
      partial_tensor_shape.AsProto(&shape_proto);
```

Let's not include spacing that is not needed.